common/sgx/sgxcertextensions.c

```diff
@@ -41,8 +41,11 @@ static const char* _tcb_comp_svn_oids[16] = {
 #define PCEID_OID SGX_EXTENSION_OID "\x03"
 #define FMSPC_OID SGX_EXTENSION_OID "\x04"
 #define SGX_TYPE_OID SGX_EXTENSION_OID "\x05"
-#define OPT_DYNAMIC_PLATFORM_OID SGX_EXTENSION_OID "\x06"
-#define OPT_CACHED_KEYS_OID SGX_EXTENSION_OID "\x07"
+#define PLATFORM_INSTANCE_ID_OID SGX_EXTENSION_OID "\x06"
```

Comment:
This field is optional. Prefix with "OPT_" to be consistent.

Reply:
Updated.